docs/model-monitoring/monitoring-models-grafana.ipynb

```diff
@@ -17,36 +17,43 @@
-    "**Project** dropdown. The Overview dashboard displays the number of endpoints in the project, the average predictions per second (using a 5-minute rolling average),\n",
-    "the average latency (using a 1-hour rolling average), and the total error count in the project.\n",
-    "\n",
-    "Additional details include:\n",
-    "* **Endpoint ID** &mdash; the ID of the deployed model. Use this link to drill down to the model performance and details panes.\n",
-    "* **Function** &mdash; the MLRun function to access the model\n",
-    "* **Model** &mdash; user defined name for the model\n",
-    "* **Model Class** &mdash; the implementation class that is used by the endpoint\n",
-    "* **First Request** &mdash; first request for production data\n",
-    "* **Last Request** &mdash; most recent request for production data\n",
-    "* **Error Count** &mdash; includes prediction process errors such as operational issues (for example, a function in a failed state), as well as data processing errors (for example, invalid timestamps, request ids, type mismatches etc.)\n",
-    "* **Drift Status** &mdash; no drift (green), possible drift (yellow), drift detected (red)\n",
-    "\n",
-    "At the bottom of the dashboard are heat maps for the Predictions per second (5 minute average), Average Latency (one hour average), and Errors.\n",
-    "See [How to Read a Heat Map](#how-to-read-a-heat-map) for more details.\n",
+    "**Project** dropdown. \n",
+    "\n",
+    "The top row of the Overview dashboard is a summary of:\n",
+    "* The number of endpoints in the project\n",
+    "* The average predictions per second (using a 5-minute rolling average),\n",
+    "* The average latency (using a 1-hour rolling average)\n",
+    "* The total error count in the project during the selected time period. \n",
+    "\n",
+    "The central table presents details about the endpoints in the project:\n",
+    "* **Name** &mdash; the model endpoint name.\n",
+    "* **Endpoint ID** &mdash; the model endpoint ID. Press it to drill down to the [performance details](#model-monitoring-details-dashboard).\n",
+    "* **Function** &mdash; the MLRun function to access the model.\n",
+    "* **Model Class** &mdash; the implementation class that is used by the endpoint (e.g., `SKLearnModelServer`).\n",
+    "* **Error Count** &mdash; the total number of errors: includes prediction process errors such as operational issues (for example, a function in a failed state), as well as data processing errors (for example, invalid timestamps, request ids, type mismatches etc.).\n",
+    "* **Sampling Percentage** &mdash; the sampling rate percentage. By default (100), all events are sampled.\n",
+    "* **Drift Status** &mdash; no drift (green), possible drift (yellow), drift detected (red).\n",
+    "\n",
+    "The graphs at the bottom are heat maps for:\n",
+    "* Predictions\n",
+    "* Average Latency\n",
+    "* Errors\n",
```

Comment:
Unlike predictions and average latency, `Errors` is not a heatmap chart. It's a time series chart that display errors by model endpoint by time.